instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/net/NetClientAttributesGetter.java

```diff
@@ -20,29 +20,23 @@
   @Nullable
   String transport(REQUEST request, @Nullable RESPONSE response);
 
+  // TODO: peerName and peerPort should be extracted onStart
+
   @Nullable
   String peerName(REQUEST request, @Nullable RESPONSE response);
 
   @Nullable
   Integer peerPort(REQUEST request, @Nullable RESPONSE response);
 
   @Nullable
-  default String sockPeerAddr(REQUEST request, @Nullable RESPONSE response) {
-    return null;
-  }
+  String sockFamily(REQUEST request, @Nullable RESPONSE response);
```

Comment:
these defaults were intentional: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/6268#discussion_r915326144

though I'm ok removing them too

Reply:
Yeah, I even liked that comment 😂 

I initially removed these defaults in this PR to make sure I implement them correctly everywhere, but I suppose it's true that they are an advanced scenario. I'll revert to `default` 👍